CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -2341,7 +2336,27 @@ + (void)showTopicsDialog:(UIWindow *)targetWindow {
             });
         }];
     }];
-    
+}
+
+#pragma mark - get the topics id if it is already exist in the userdefault
++ (BOOL)selectedTopics{
+    NSArray* topics = [self getSubscriptionTopics];
+    if (!topics || [topics count] == 0) {
```

Comment:
if the array is not null and count of it is 0, then you return true for method name "hasSelectedTopics"?
also, pragma mark comment is misleading. you're not getting any topic id inside or am i wrong?

you're checking if topics array exists? and if its count is 0, as i can read out of it.


also if the above is intended/correct you can just do "return !topics || [topics count] == 0". no need for else return true / false-

also shouldnt the method name be hasSelectedTopics

Reply:
Ok I will modified the things and commit a new code.